### ISSUE TITLE
Deleting a sensitive config var renders the actual value in terraform plan output

### ISSUE THREAD

**Issue by sheax0r** [terraform-providers/terraform-provider-heroku#194]
### Terraform Version
Run `terraform -v` to show the version. If you are not running the latest version of Terraform, please upgrade because your issue may have already been fixed.

### Heroku Provider Version
```
Terraform v0.11.13
+ provider.heroku v1.9.0
```

### Affected Resource(s)
* heroku_app

### Terraform Configuration Files
```hcl
variable "app_name" {}

variable "config_var_value" {}

resource "heroku_app" "main" {
  region = "us"
  name   = "${var.app_name}"

  sensitive_config_vars {
    KEY = "${var.config_var_value}"
  }
}
```

### Expected Behavior
When deleting a value from `sensitive_config_vars` and running `terraform plan`, the sensitive value should be displayed as `<sensitive>`

### Actual Behavior
The sensitive value was actually displayed as follows when running `terraform plan`:

```
  ~ heroku_app.main
      sensitive_config_vars.0.%:   "1" => "0"
      sensitive_config_vars.0.KEY: "value1" => ""
```

### Steps to Reproduce
1. Create main.tf using the given HCL file above.
2. Export `TF_VAR_app_name` to a unique name, to create your own heroku app, and `TF_VAR_config_var_value=value1`.
3. Run `terraform apply` to create the app.
4. Comment out the `KEY = "${var.config_var_value}"` line in the `sensitive_config_vars` block.
5. Run `terraform plan`. Note that the to-be-deleted sensitive config var is rendered.

**Comment by davidji99** [comment]
Hey @sheax0r,

So this is a limitation in terraform-core in that deletes aren't respecting the `Sensitive` attribute in the resource's schema. I asked Hashicorp about this and they mentioned that it 'should' be in fixed in 0.12.0.

**Comment by sheax0r** [comment]
@davidji99 Cool -- It's okay with me if this gets closed, then. Do as you wish. Thanks!

**Comment by davidji99** [comment]
Going to leave this issue open incase others have the same issue/concern as you did.

**Comment by davidji99** [comment]
This should be fixed once #179 goes in. Here's how the output looks when I delete a sensitive_config_var:

```
  # heroku_app_config_association.foobar will be updated in-place
  ~ resource "heroku_app_config_association" "foobar" {
        app_id         = "some_app"
      ~ sensitive_vars = (sensitive value)
        vars           = {}
    }
```

**Comment by davidji99** [comment]
Closing this issue @sheax0r as v2 was recently released.
